templates/vhost.conf

```diff
@@ -2,12 +2,18 @@
 
 <VirtualHost *:{{ port }}>
         ServerName {{ _website_domain }}
-    {% if _force_tls and port|int == 80 %}
+    {% if (_force_tls and port|int == 80) or reverse_proxy is defined %}
         RewriteEngine On
         # for lets encrypt automation
         RewriteCond %{REQUEST_URI} !^/.well-known/acme-challenge/
-        RewriteCond %{HTTPS} off
-        RewriteRule (.*) https://%{HTTP_HOST}%{REQUEST_URI}
+
+        {% if _force_tls %}
+          RewriteCond %{HTTPS} off
+          RewriteRule (.*) https://%{HTTP_HOST}%{REQUEST_URI}
+        {% else if reverse_proxy is defined %}
+          RewriteRule ^/(.*)         {{ reverse_proxy }}$1 [L,P]
```

Comment:
Indeed. I need to verify the semantic of reverse_proxy, and in the end, we do not even need the $1 and the regexp capture, since the data is already there in REQUEST_URI. We can surely avoid using 10K of memory per request, which is gonna have no effect whatsoever in practice :)